src/Models/Scopes/Attribute/OnlyProductAttributesScope.php

```diff
@@ -45,10 +44,15 @@ public function apply(Builder $builder, Model $model)
                 'position'
             )
             ->join('catalog_eav_attribute', 'eav_attribute.attribute_id', '=', 'catalog_eav_attribute.attribute_id')
-            ->leftJoin('eav_attribute_label', function ($join) {
+            ->leftJoin('eav_attribute_label', function (JoinClause $join) {
                 $join->on('eav_attribute.attribute_id', '=', 'eav_attribute_label.attribute_id')
                     ->where('eav_attribute_label.store_id', config('rapidez.store'));
             })
+            ->leftJoin('information_schema.columns', function (JoinClause $join) use ($productTable, $databaseName) {
+                $join->on('eav_attribute.attribute_code', '=', 'column_name')
+                    ->where('table_name', $productTable)
+                    ->where('table_schema', $databaseName);
+            })
```

Comment:
https://dev.mysql.com/doc/mysql-infoschema-excerpt/5.7/en/information-schema-introduction.html#:~:text=to%20SHOW%20Statements.-,INFORMATION_SCHEMA%20and%20Privileges,has%20the%20proper%20access%20privileges.

Privileges seem to work fine for this, but be sure to check this thoroughly

Reply:
Laravel also uses the information schema for methods like `hasColumn ()`, etc: https://github.com/laravel/framework/blob/726c352e54d61d48204f96c72377a7b4d48afe48/src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php#L126